runtime/src/main/java/org/corfudb/protocols/CorfuProtocolServerErrors.java

```diff
@@ -0,0 +1,151 @@
+package org.corfudb.protocols;
+
+import com.google.protobuf.ByteString;
+import lombok.extern.slf4j.Slf4j;
+import java.io.ObjectOutputStream;
+
+import org.corfudb.runtime.proto.RpcCommon.UuidMsg;
+import org.corfudb.runtime.proto.ServerErrors.BootstrappedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.DataCorruptionErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.DataOutrankedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.NotBootstrappedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.NotReadyErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.OverwriteErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.ServerErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.TrimmedErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.UnknownErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.WrongClusterErrorMsg;
+import org.corfudb.runtime.proto.ServerErrors.WrongEpochErrorMsg;
+
+/**
+ * This class provides methods for creating the Protobuf objects defined
+ * in server_errors.proto and are used by several of the service RPCs.
+ */
+@Slf4j
+public class CorfuProtocolServerErrors {
+    // Prevent class from being instantiated
+    private CorfuProtocolServerErrors() {}
+
+    /**
+     * Returns the Protobuf representation of a WRONG_EPOCH error.
+     *
+     * @param correctEpoch   The correct epoch.
+     */
+    public static ServerErrorMsg getWrongEpochErrorMsg(long correctEpoch) {
+        return ServerErrorMsg.newBuilder()
+                .setWrongEpochError(WrongEpochErrorMsg.newBuilder()
+                        .setCorrectEpoch(correctEpoch)
+                        .build())
+                .build();
+    }
+
+    /**
+     * Returns the Protobuf representation of a NOT_READY error.
+     */
+    public static ServerErrorMsg getNotReadyErrorMsg() {
+        return ServerErrorMsg.newBuilder()
+                .setNotReadyError(NotReadyErrorMsg.getDefaultInstance())
+                .build();
+    }
+
+    /**
+     * Returns the Protobuf representation of a WRONG_CLUSTER_ID error.
+     * @param expectedClusterId   The cluster ID of the server.
+     * @param providedClusterId   The ID of the cluster provided by the client.
+     */
+    public static ServerErrorMsg getWrongClusterErrorMsg(UuidMsg expectedClusterId, UuidMsg providedClusterId) {
```

Comment:
missing java doc for @return ?

Reply:
Added these for CorfuProtocolServerErrors, and also the APIs from the previous PR.